src/python/pants/backend/python/target_types.py

```diff
@@ -514,27 +514,32 @@ class PexIncludeToolsField(BoolField):
     )
 
 
+_PEX_BINARY_COMMON_FIELDS = (
+    *COMMON_TARGET_FIELDS,
+    InterpreterConstraintsField,
+    PythonResolveField,
+    PexBinaryDependenciesField,
+    PexPlatformsField,
+    PexResolveLocalPlatformsField,
+    PexInheritPathField,
+    PexStripEnvField,
+    PexIgnoreErrorsField,
+    PexShebangField,
+    PexEmitWarningsField,
+    PexLayoutField,
+    PexExecutionModeField,
+    PexIncludeToolsField,
+    RestartableField,
+)
+
+
 class PexBinary(Target):
     alias = "pex_binary"
     core_fields = (
-        *COMMON_TARGET_FIELDS,
-        OutputPathField,
-        InterpreterConstraintsField,
-        PythonResolveField,
-        PexBinaryDependenciesField,
+        *_PEX_BINARY_COMMON_FIELDS,
         PexEntryPointField,
         PexScriptField,
-        PexPlatformsField,
-        PexResolveLocalPlatformsField,
-        PexInheritPathField,
-        PexStripEnvField,
-        PexIgnoreErrorsField,
-        PexShebangField,
-        PexEmitWarningsField,
-        PexLayoutField,
-        PexExecutionModeField,
-        PexIncludeToolsField,
-        RestartableField,
+        OutputPathField,  # TOREVIEW: I don't think this makes sense on `pex_binaries_...``
```

Comment:
You're right, it definitely does not make sense. That should only be set via the `overrides` field